### PR TITLE
Implemented DateModuleResult

### PR DESCRIPTION
**Pull request by danielkza**
Signed off by: Eduardo Silva <duduktamg@hotmail.com>

### CODE REVIEW

File: kalibro_client/miscellaneous/date_module_result.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from recordtype import recordtype
+
+class DateModuleResult(recordtype('DateModuleResult', 'date module_result')):
```
Comment (marcheing) [inline kalibro_client/miscellaneous/date_module_result.py]: Maybe we could use a tuple were to keep the standard

File: tests/factories.py [hunk 2]
```diff
@@ -120,3 +121,11 @@ class Meta:
     grade = 10
     color = "33dd33"
     reading_group_id = 1
+
+class DateModuleResultFactory(factory.Factory):
```
Comment (marcheing) [inline tests/factories.py]: You guys forgot to close the slots

File: kalibro_client/miscellaneous/date_module_result.py (hunk 1 above)
Comment (danielkza) [inline kalibro_client/miscellaneous/date_module_result.py]: OK

File: tests/factories.py (hunk 2 above)
Comment (danielkza) [inline tests/factories.py]: OK

**Pull request merged**